### PR TITLE
Fixes for colorpicker on firefox

### PR DESCRIPTION
**Pull request by dckesler**

### CODE REVIEW

File: src/custom-elements/cps-colorpicker/slider.component.js [hunk 1]
```diff
@@ -20,14 +20,15 @@ export default class ColorSlider extends Component {
...
-					this.xStart = e.x;
+					this.xStart = e.pageX;
 					this.setState({
 						dragging: true,
 					})
 					window.addEventListener('mousemove', this.mousemove);
 					window.addEventListener('mouseup', this.mouseup);
 				}}
 				style={{
+					top: `-${(props.sliderSize / 2) - (props.barHeight / 2)}px`,
```
Comment (dckesler) [inline src/custom-elements/cps-colorpicker/slider.component.js]: Absolute positioned elements in Firefox don't get flex positioning. So I have to manually center it vertically.

File: .fontcustom-manifest.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "checksum": {
-    "previous": "1cea44b5c48c6b7d0b8f500d87cf05faaa2b909fcf73838c1f1327cccd011e5d",
-    "current": "1cea44b5c48c6b7d0b8f500d87cf05faaa2b909fcf73838c1f1327cccd011e5d"
+    "previous": "65650da80411b2cbadb36c425b50f3a37573095925a57eb0d46c4848ecfbc3f1",
```
Comment (kentmclean) [inline .fontcustom-manifest.json]: are these `fontcustom-manifest` value changes intentional?

Comment (dckesler) [inline .fontcustom-manifest.json]: No

Comment (dckesler) [inline .fontcustom-manifest.json]: Just happened on install

**Pull request merged**